frontend/src/pages/my-projects/components/components.ts

```diff
@@ -1,2 +1,3 @@
-export { CreateProjectModal } from './create-project-form/create-project-modal.js';
+export { CreatePopup } from './create-popup/create-popup.js';
 export { ProjectCard } from './project-card/project-card.js';
+export { ProjectModal } from './project-modal/project-modal.js';
```

Comment:
you should have only one component for it. let's name it as `configurate-project-popup`